server/src/main/java/io/druid/initialization/Initialization.java

```diff
@@ -275,6 +278,41 @@ public static URLClassLoader getClassLoaderForExtension(File extension) throws M
     return loader;
   }
 
+  public static List<URL> getURLsForClasspath(String cp)
+  {
+    try {
+      String[] paths = cp.split(File.pathSeparator);
+
+      List<URL> urls = new ArrayList<>();
+      for (int i = 0; i < paths.length; i++) {
+        File f = new File(paths[i]);
+        if ("*".equals(f.getName())) {
```

Comment:
My concern here is that the classpath parsing is not done using java internal parsing stuff, so someone could have a classpath as an argument for their java command line, and specify some other classpath for the argument here, and end up with different sets of stuff.

Is there a way this classpath parsing can be done using the same parsing as the jvm?